test/examples/ucp_hello_world.c

```diff
@@ -329,6 +332,14 @@ static int run_ucx_server(ucp_worker_h ucp_worker)
         goto err;
     }
 
+    test_str = (char*) calloc(test_string_length + 1, sizeof(char));
```

Comment:
@guy101 I assume the user asks for 256 **bytes**, not the 256 **characters**. In this case, you have to provide 255 **characters** + 1 zero terminator = 256 **bytes**.